lib/marcel/tables.rb

```diff
@@ -2494,8 +2495,8 @@ module Marcel
     ['message/rfc822', [[0..1000, "\nMessage-ID:"]]],
     ['text/html', [[0..64, '<!DOCTYPE HTML'], [0..64, '<!DOCTYPE html'], [0..64, '<!doctype HTML'], [0..64, '<!doctype html'], [0..64, '<HEAD'], [0..64, '<head'], [0..64, '<TITLE'], [0..64, '<title'], [0..64, '<HTML'], [0, '<BODY'], [0, '<body'], [0, '<DIV'], [0, '<div'], [0, '<TITLE'], [0, '<title'], [0, '<h1'], [0, '<H1'], [0..128, '<html']]],
     ['text/vtt', [[0, "WEBVTT\r"], [0, "WEBVTT\n"], [0, '0xfeff', [[2, "WEBVTT\r"]]], [0, '0xfeff', [[2, "WEBVTT\n"]]], [0, "WEBVTT FILE\r"], [0, "WEBVTT FILE\n"]]],
-    ['text/x-matlab', [[0, "function [a-zA-Z][A-Za-z0-9_]{0,62}\\\\s*="]]],
     ['text/x-matlab', [[0, "function [a-zA-Z][A-Za-z0-9_]{0,62}[\\\\r\\\\n]"]]],
+    ['text/x-matlab', [[0, "function [a-zA-Z][A-Za-z0-9_]{0,62}\\\\s*="]]],
```

Comment:
You don't mention any reason for this change - is it just part of the process building the file?

Reply:
Yes, I'll double check before merging. `bundle exec rake tables` regenerates this code from reading the XML.

Reply:
The two rules that get switched around are both the same priority. https://github.com/rails/marcel/blob/85c2559ac0253b9b33cae10c36e3f2584efe3991/data/tika.xml#L7136-L7144

The `application/x-tika-msoffice` magic priority I'm overriding is priority 40, and using 41-50 works, but also shuffles around other rules that don't seem to matter (because they have the same priority). I think it is related to how we [sort the magic rules](https://github.com/rails/marcel/blob/85c2559ac0253b9b33cae10c36e3f2584efe3991/script/generate_tables.rb#L124) and alpha sort.